README.md

```diff
@@ -10,48 +10,38 @@ Create a `.eslintrc.json` file in the project root containing the following:
 
 ```json
 {
-  "extends": ["warp"],
-  "env": {
-    "browser": true
-  }
+  "extends": ["warp"]
 }
 ```
 
 Always [configure your environment](http://eslint.org/docs/user-guide/configuring#specifying-environments).
+We recommend doing so by using the appropriate [preset](#presets):
 
-**Inside NodeJS** (`"env":{"node":true}` is implied)
-
-```console
-$ npm install --save-dev eslint-plugin-node
-```
+## Presets
 
-```json
-{
-  "extends": ["warp/node"]
-}
-```
-
-**With ES6 support** (`"env":{"es6":true}` is implied)
-
-```json
-{
-  "extends": ["warp/es6"]
-}
-```
+This package comes with some additional presets to include in your eslint
+configuration. To include any of them, just add them to the `extends` Array.
+Note that order is important, because each preset overrides the previous. To
+allow you to configure a working load order, the table below provides hints.
 
-**ES6 module**
+Preset | Description | Load Order | Dependencies
+-------|-------------|------------|--------------
+`warp/node` | Linting rules for Node.JS | Soon after `warp` | `eslint-plugin-node`
+`warp/es6` | Linting rules for ES6 language features | Soon after `warp` |
+`warp/jsx` | Linting rules for React JSX | Soon after `warp` | `eslint-config-standard-react eslint-plugin-react`
+`warp/module` | Linting rules for modular JavaScript | Near the end |
```

Comment:
If `warp/module` no longer extends `warp/es6`, does it now not have a dependency on it?

Reply:
You'd probably use them together most of the time, but I don't think there's a hard dependency.